server/apps/main/templates/main/experiences_page.html

```diff
@@ -52,12 +53,11 @@ <h1 class="big-heading">View Stories</h1>
             <input class="form-control mb-2" id="inlineFormInput" placeholder="Search Stories" value="{{ searched }}" type="text" name="searched" />
           </div>
           <div class="col-md-2">
-            <button class="btn btn-primary mb-2" type="submit">Search Stories</button>
+            <button class="btn btn-primary mb-2" type="submit">Search stories</button>
           </div>
         </div>
-        <div class="trigger-warning-message"> We have excluded potentially
-          triggering content, please select any categories that you would be
-          comfortable seeing
+        <div class="trigger-warning-message">
+          We have excluded potentially triggering content. Please select any categories you would be comfortable seeing:
```

Comment:
Is it worth putting this in bold or making it stand out in some way given lots of people in user testing glanced over it and got it the wrong way round?

Reply:
I made it bold!